packages/core/src/Button/Button.styles.ts

```diff
@@ -67,27 +72,70 @@ export const { staticClasses, useClasses } = createClasses("HvButton", {
   },
   primarySubtle: {
     color: theme.colors.primary,
-    backgroundColor: "transparent",
-    border: "1px solid currentColor",
+    ...subtle,
   },
   primaryGhost: {
     color: theme.colors.primary,
-    backgroundColor: "transparent",
-    "&$disabled": {
-      backgroundColor: "transparent",
+    ...ghost,
+  },
+  positive: {
+    color: theme.colors.atmo1,
+    backgroundColor: theme.colors.positive,
+    "&:not($disabled)": {
+      "&:hover, &:focus-visible": {
+        backgroundColor: theme.colors.positive_80,
+      },
     },
   },
+  positiveSubtle: {
+    color: theme.colors.positive,
+    ...subtle,
+  },
```

Comment:
There is lots of duplication here (that we can remove) - and users will get all of this in their bundles whether they use these new variants of not...

My suggestions are:
- Keep the disabled "overrides" in the disabled class (increase `backgroundColor` specificity)
- Have classes for the button types (eg. `subtle`/`ghost`/`contained`?) and apply the styles there
- Do the "coloring" (`color` + `backgroundColor`) with the `getX` utilities like we're doing for the sizes